manager/README.md

```diff
@@ -8,48 +8,109 @@ The `manager` binary includes all of the controllers that this project defines b
 - `enable-blueprint-controller` to enable the controller for `Blueprint`
 - `enable-motion-controller` to enable the controllers for `BatchTransfer` and `StreamTransfer`
 
+
+## Run and debug locally
+
+Beyond testing, you may run and debug the manager outside the cluster using the following instructions.
+
+### Run required components
+
+Components such as connectors need to be running before you run the manager.
+This can be done by one of these options:
+1. Running components locally directly (no instructions provided)
+2. Running components in a cluster 
+
+For option 2, the Helm installation allows you to pick which compoenents to install. 
+Follow the installation guide as usual but in the Helm installation for the control plane add `--set manager.enabled=false` to skip the deployment of the manager. For example:
+
+```bash
+helm install m4d charts/m4d --set global.tag=latest --set manager.enabled=false -n m4d-system --wait
+```
+
+### Expose running components
+
+Components such as connectors need to be reachable over localhost.
+If you chose to run these components in a cluster you can use port-forward.
+For example:
+
+```bash
+kubectl -n m4d-system port-forward svc/katalog-connector 49152:80 &
+kubectl -n m4d-system port-forward svc/opa-connector 49153:80 &
+```
+
+### Set configuration environment variables
+
+The main configuration map is not available when running locally.
+Therefore, you need to define configuration as environment variables.
+
+Create `.env` file in the root folder of the project. For example:
+
+```bash
+VAULT_ADDRESS="http://vault.m4d-system:8200"
+MAIN_POLICY_MANAGER_NAME="opa"
+MAIN_POLICY_MANAGER_CONNECTOR_URL="localhost:49153"
+CATALOG_PROVIDER_NAME="katalog"
+CATALOG_CONNECTOR_URL="localhost:49152"
+CONNECTION_TIMEOUT="120"
+USE_EXTENSIONPOLICY_MANAGER="false"
+VAULT_MODULES_ROLE="module"
+ENABLE_WEBHOOKS="false"
+```
+
+If you plan to run manager from the command line,
+then run the following to export all of the variables:
+
+```bash
+export $(xargs < ./.env)
+```
+
+### Run the manager
+
+### From command line
+
+You can now run the manager from the `manager` folder using one of these options:
+1. `make run`
+2. `go run main.go --enable-all-controllers --metrics-bind-addr=0`
+
```

Comment:
That did not work for me (both options) I got "missing go.sum entry for module providing" errors